### ISSUE TITLE
A11y_Windows Template Studio_Select a Project type_Keyboard: When we move to selected list Item Keyboard focus is not visible on it.

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/WindowsTemplateStudio#3900]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#GH_WindowsTemplateStudio_Oct2020; #Windows Template Studio;#FTP;#DesktopApp;#WCAG2.4.7;#NonBenchmark;#keyboard;#FocusVisible;#Win10;

### **Environment Details:**
Application Name: Windows Template Studio (WinTS)
Templates version: 3.8.20273.1
Wizard version: 3.8.20273.1Date: 2020-06-17T21:13:20.174Z
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.7.0 Preview 6.0
Windows Version: Windows10

### **Repro Steps:**
1.	Open Visual Studio 2019 
2.	Click on File from menu and then select new and then project.
3.	Select the "Windows template studio (WPF.Net Core)”. Click on next
4.	"Configure your new project" screen will appear. Click on create button.
5.	New WPF App screen will open.
6.	Navigate to “Project Type” List item present in Navigation panel with the help of Tab key.
7.	Check keyboard focus is visible or not on selected list item “Project Type”

### **Actual:**
When we move to selected list Item(Project Type) , Keyboard focus outline is not visible on it.

**Same issue is also observed with list items present in Following Scenarios:** 
•	Select a Design pattern
•	Add Pages
•	Add Features
•	Add Services
•	Select the Template You Want To Add
•	Template Details
•	Change Summary

### **Expected:**
Keyboard focus should be clearly visible on Selected list Item.

### **User Impact:**
Keyboard users will not be able to see where is the keyboard focus if focus is not visible properly.

**Comment by harshgangrade111** [comment]
@sibille @vmadurga Issue is still repro in the  latest dev-nightly version. Keyboard focus outline is not visible on selected list item when we move back on it.

**Comment by sibille** [comment]
@harshgangrade111, this is not fixed yet, we're waiting for a design-review for our proposed solution. 
Once we fixed the issue, I'll add the done tag then and reassign to you to test and close.

**Comment by sibille** [comment]
Spoke to the design team and they suggested to use the FocusVisualStyle from ThemedDialog here instead of our proposed solution.

**Comment by sibille** [comment]
Verified in dev-nightly version
Wizard version: 0.22.20335.2

You can install the dev-nightly version from our nightly feed (https://www.myget.org/F/windows-template-studio-nightly/vsix/).
For instructions on how install WTS from the dev-nightly feed in Visual Studio see https://github.com/microsoft/WindowsTemplateStudio/blob/dev/docs/getting-started-extension.md#nightly--pre-release-instructions

**Comment by harshgangrade111** [comment]
@sibille issue is still repro in the latest Wizard version: 0.22.20337.1.
keyboard focus outline is not visible clearly on selected list item present in left navigation. Color contrast ratio of selected list item and color contrast ratio keyboard focus outline should be greater than or equal to 3:1.

**Comment by sibille** [comment]
@JeanRoca , this is the issue I mentioned earlier. 
Problem here is that the contrast between the selected item and the focus outline in not sufficient. 
![image](https://user-images.githubusercontent.com/24698018/101667632-daa4e880-3a4f-11eb-8ca4-dd0708abed07.png)



We're working on changing the colors on the left navigation menu to colors from `ThemedDialog` as suggested by the Design Team. This will ensure that the contrast ratio is met.

**Comment by JeanRoca** [comment]
@sibille perfect thank you for the update

**Comment by sibille** [comment]
Verified in dev-nightly version
Templates version: 0.22.20345.2
Wizard version: 0.22.20345.2

![image](https://user-images.githubusercontent.com/24698018/101784230-87886f80-3afb-11eb-956d-a79ae7d65dd7.png)

**Comment by sibille** [comment]
@harshgangrade111, this should be fixed now, can you verify and close?

**Comment by harshgangrade111** [comment]
Issue does not repro in the latest Wizard version: 0.22.20349.1. Hence closing this bug.
